ansible/roles/app/templates/support.conf.j2

```diff
@@ -8,51 +8,25 @@ Listen {{ apache_app_listen_address.support }}:{{ loadbalancing.support.port }}
     ServerAdmin {{ apache_server_admin }}
     Options -indexes +Includes
     DocumentRoot {{ document_root }}
```

Comment:
I'd like to change the var {{ document_root }} to something less generic and more SP dashboard specific, since this Ansible role will be used from a playbook in OpenConext-deploy as well.